events/diskpersist.go

```diff
@@ -443,7 +447,16 @@ func (dp *DiskPersistence) doPersist(ctx context.Context, j persistJob) error {
 	b := j.Bytes
 	e := j.Evt
 	seq := dp.curSeq
-	dp.curSeq++
+	if dp.timeSequence {
+		nextSeq := time.Now().UnixMicro()
```

Comment:
scroll down about 40 lines to discover why this may break the relay

Reply:
right. will need to keep a separate event counter if seq isn't it, and the playback logic needs some touchup too

Reply:
fixed fragmentation via event counter, and fixed replay